### ISSUE TITLE
ActiveIndex on PageControl does not [fully] work

### ISSUE THREAD

**Issue by eksperts** [fusetools/fuselibs-public#189]
[Reported on community](https://www.fusetools.com/community/forums/bug_reports/pagecontrol_activeindex_bug). Affects all previews and Android build at least too. Tested on macOS + Fuse 1.0.3 and Fuse 1.1.0.

The Blue page should be active when launching the app, but the Red one is shown instead:
```
<App>
    <ClientPanel>
      <PageControl ActiveIndex="1">
          <Panel Color="Red"></Panel>
          <Panel Color="Blue"></Panel>
          <Panel Color="Yellow"></Panel>
      </PageControl>
    </ClientPanel>
</App>
```
If you change the `ActiveIndex` value to `2` and save the file, `PageControl` correctly switches to the Yellow page. If you then hit `CMD+R` in Studio, it reverts back to showing the Red page.

**Comment by fusebuild** [comment]
The forum thread [PageControl ActiveIndex bug](https://www.fusetools.com/community/forums/bug_reports/pagecontrol_activeindex_bug) posted by prince was linked to this issue.

**Comment by kusma** [comment]
This seems to have been fixed at some point before 1.3
